### ISSUE TITLE
Accounts with captcha get "locked" if the process is killed before validate captcha

### ISSUE THREAD

**Issue by rodrgomes** [RocketMap/RocketMap#2267]
I was scanning a single location with a fresh account and it got a captcha. I stopped the process without validate it.
After I restarted, the account were getting 0/0/0 without asking for captcha. Tried use that account in game and the first thing the game asked after logon was captcha. After I entered capcha (in game) the account was normal again, even in RM.

Seems that RM isn't detecting the captcha in logon, only if the account is asked for captcha when its already logged.

## Expected Behavior
After you start RM it should ask for captcha if detected in login

## Current Behavior
Every time a captched account try scan it always get 0/0/0

## Possible Solution
Detect captcha in login and put account in "Accounts with captcha"

## Steps to Reproduce (for bugs)
1. Wait to some account get  a captcha
2. Write down the account name, and kill the RM 
3. Try use that account again

**Comment by friscoMad** [comment]
Confirmed it does send the captcha during the get config and get assets messages but not anymore after that, I left it running and after 180 GMOs there was no other message with captcha_url.

**Comment by farstarss** [comment]
I noticed the same today and just kept waiting. Captchas showed up about 40 to 70 after the new instance with the 'locked' accounts was started. Solving those solved my problem.

I assume captchas are not detected during login but upon refreshing the session or soon after.

**Comment by friscoMad** [comment]
yeah maybe it was when refreshing the session, I tried to find a way to handle the captchas properly during login but RM is so coupled that it needs a lot of refactoring to be added (or a lot of extra params needs to be added to a lot of places 😢 ).
